### PR TITLE
Introduce @enum LWC_PRICE_SCALE_MODE

### PR DESCRIPTION
**Pull request by g-gundam**
### Pull request checklist

- [x] Did you bump the project version?
- [x] Did you add a description to the Pull Request?
- [x] Did you add new tests?
- [ ] Did you add reviewers?

### Backstory

In order to visualize multi-year BTCUSD charts effectively, I needed to be able to change the price axis on the charts to log scale.  To do this, I did the following:
- I introduced a new enum LWC_PRICE_SCALE_MODE that corresponds to [PriceScaleMode](https://tradingview.github.io/lightweight-charts/docs/api/enums/PriceScaleMode) from LightweightCharts.js.
- I updated the lwc_panel method and the LWCPanel struct to include `mode` which should be a LWC_PRICE_SCALE_MODE value.
- I added a Serde.SerJson.ser_type method for LWC_PRICE_SCALE_MODE.
- I modified components/chart_panel.jsx to make use of this new `mode` value.
- I regenerated index_boundle.js.
- I slightly updated test/panel.jl to use a constant from LWC_PRICE_SCALE_MODE.

I tried to follow the conventions I observed, but I'm a bit unsure if I did things correctly.  However, I figured I'd share my first attempt, and get some feedback.

#### Linear Scale (default)

linear-scale

#### Logarithmic Scale (new)

log-scale

It's the same data in both charts, but log scale makes the big value changes in BTCUSD more manageable.

### CODE REVIEW

File: src/LightweightCharts.jl [hunk 1]
```diff
@@ -193,6 +200,7 @@ function lwc_panel(
     tooltip::Bool = true,
     tooltip_format::String = "\${label_name}: (\${time}, \${value})",
     min_charts_for_search = 10,
+    mode = LWC_NORMAL
```
Comment (gryumov) [inline src/LightweightCharts.jl]:
```suggestion
    mode::LWC_PRICE_SCALE_MODE = LWC_NORMAL
```

File: src/LightweightCharts.jl [hunk 2]
```diff
@@ -139,6 +144,7 @@ mutable struct LWCPanel <: AbstractChartSettings
     copyright::Bool
     tooltip::Bool
     tooltip_format::String
+    mode::Any
```
Comment (gryumov) [inline src/LightweightCharts.jl]:
```suggestion
    mode::LWC_PRICE_SCALE_MODE
```

File: src/LightweightCharts.jl [hunk 3]
```diff
@@ -193,6 +200,7 @@ function lwc_panel(
     tooltip::Bool = true,
     tooltip_format::String = "\${label_name}: (\${time}, \${value})",
     min_charts_for_search = 10,
```
Comment (gryumov) [inline src/LightweightCharts.jl]:
```suggestion
    min_charts_for_search::Int64 = 10,
```

File: src/LightweightCharts.jl [hunk 4]
```diff
@@ -57,6 +57,11 @@ export LWC_TOOLTIP_TYPE,
     LWC_TOOLTIP_TOP,
     LWC_TOOLTIP_TRACK
 
+export LWC_NORMAL,
+    LWC_LOGARITHMIC,
+    LWC_PERCENTAGE,
+    LWC_INDEXED_TO_100
```
Comment (gryumov) [inline src/LightweightCharts.jl]:
```suggestion
export LWC_PRICE_SCALE_MODE,
    LWC_NORMAL,
    LWC_LOGARITHMIC,
    LWC_PERCENTAGE,
    LWC_INDEXED_TO_100
```

**Comment by gryumov** [comment]
@g-gundam, thanks for your PR; I made a couple of suggestions, but everything else looks good.

**Comment by g-gundam** [comment]
All of your suggestions seemed reasonable to me, so I went ahead and confirmed those changes.

**Review by gryumov — APPROVED** [review]

**Pull request merged**